cdap-docs/developers-manual/source/building-blocks/applications.rst

```diff
@@ -51,6 +51,38 @@ name.
 Names used for streams and datasets need to be unique across the CDAP instance, while
 names used for programs and services need to be unique only to the application.
 
+Application can also use a Configuration class to get a configuration during deployment time of the Application.
+This can be used to, for example, provide the stream, dataset to create/read from during application deployment time
```

Comment:
For example, this can be used to specify|---|at application deployment time|---|a stream to be created or a dataset to be read instead of hard-coding it in... [delete "For example, "] We can modify...